apps/web/src/pages/Applications/ApplicationWelcomePage/ApplicationWelcomePage.tsx

```diff
@@ -118,13 +119,21 @@ const ApplicationWelcome = ({ application }: ApplicationPageProps) => {
         )}
       </p>
       <p data-h2-margin="base(x1, 0)">
-        {intl.formatMessage({
-          defaultMessage:
-            "The GC Digital Talent platform is a skills-based hiring system. This means that your application will put a heavier focus on your skills and how you've used them in past experiences to help us get a stronger understanding of your fit.",
-          id: "u/DBSl",
-          description:
-            "Description of how the skills-based hiring platform assess candidates.",
-        })}
+        {isIAP
+          ? intl.formatMessage({
+              defaultMessage:
+                "The program is a Government of Canada initiative specifically for First Nations, Inuit, and Métis peoples. It is a pathway to employment in the federal public service for Indigenous peoples who have a passion for Information Technology (IT). We focus on that passion, and your potential to grow and succeed in this field.",
+              id: "rJLUje",
+              description:
+                "Description of how the skills-based hiring platform assess candidates.",
```

Comment:
This description is exactly the same as another string that is not the same. Don't forget to update the id and description in fr.json as this change will modify its id and description.
```suggestion
                "Description of how the hiring platform assesses candidates for IAP.",
```